product_barcode_generator/models/product.py

```diff
@@ -36,21 +36,20 @@ class ProductCategory(models.Model):
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
-    ean13 = fields.Char(copy=False)
     ean_sequence_id = fields.Many2one('ir.sequence', string='Ean sequence')
```

Comment:
Check _get_ean_next_code -- it allows to use different sequence for the product. But it's strange to have a separate sequence on a product level. 
This PR is intended to port module rather than improve its functionality